IET-DAVV/website-frontend
# Kamna

## **Title: Implement Contact Section with Google Maps Embed and Form**

### **Description**
- This PR adds a new "Contact Us" section to the website. It includes an embedded Google Maps widget, contact details (address, email, phone), and a user-friendly message form that allows visitors to submit inquiries directly through the site.
- The contact section enhances user interaction by providing clear contact information and a convenient form for visitors to send messages directly from the website, improving the user experience and accessibility.

### **Proposed Changes**
- Integrated Google Maps to show the physical location of the institution, enhancing user trust and engagement.
- Styled the form and contact details using consistent fonts and spacing for a professional, clean look.

### **Screenshots **
![Screenshot 2024-09-10 000223](https://github.com/user-attachments/assets/7210afdd-4c59-480f-86c1-c25ce5bed802)


### **Testing**
- **Manual Testing:** 
      Verified that the contact details (address, email, phone) are correctly displayed.
      Confirmed that the Google Maps embed is functional and correctly centered.
      Tested the message form to ensure all input fields work properly and that the submit button is responsive.

### **Steps to Test**
-Clone the repository and pull the latest changes.
-Navigate to the "Contact Us" section on the website.
-Verify that:
    The contact details (address, email, phone) are visible and accurate.
    Google Maps is properly loaded and displays the correct location.
    The contact form can accept input in all fields and the submit button responds as expected.

### **Checklist**


---
